### ISSUE TITLE
Transformer hugginface BERT model not working 

### ISSUE THREAD

**Issue by bksaini078** [apple/tensorflow_macos#271]
While running ```transformers.TFDistilBertModel.from_pretrained(pretrained_weights)```
it showing the error message.
![image](https://user-images.githubusercontent.com/53876680/119178491-1962b400-ba6e-11eb-85b8-63a3af9cff26.png)
Can someone please help how to resolve this issue?  Or, someone able to run the Transfomer BERT models in mac M1?

**Comment by ww232330** [comment]
change layer to
x=tf.keras.layers.Dropout(0.2)(x)
x= tf.keras.layers.Dense(64)(x)
x=tf.keras.layers.Dense(32)(x)
x=tf.keras.layers.Dense(2,activation='sigmoid')(x)
output=tf.keras.layers.Dropout(0)(x)

**Comment by bksaini078** [comment]
Thank you for your reply, 
I tried the proposed approach. Unfortunately, it is showing the same error message. 
Did you run the BERT model successfully on your end?

**Comment by jaismith** [comment]
hey there @bksaini078, were you able to load the BERT from `tensorflow-hub`? if so, would you mind showing how you did that? i'm unable to load the BERT model using `hub.KerasLayer` (see #276)
